control_panel_api/serializers.py

```diff
@@ -80,3 +81,22 @@ class S3BucketSerializer(serializers.ModelSerializer):
     class Meta:
         model = S3Bucket
         fields = ('id', 'url', 'name', 'arn', 'apps3buckets', 'created_by')
```

Comment:
Shouldn't we add/expose the `userapps` association here and in the `AppSerializer` for consistency?

Reply:
Speaking of serialisers and associations, `S3BucketSerializer` should also expose `users3buckets` (currently only expose `apps3buckets`.
Basically, we should be able to get these "through" associations from all the serialisers for `App`, `S3Bucket` and `User`.